### PR TITLE
Add phphelpfile.md documentation

### PR DESCRIPTION
**Pull request by DavidLRowe**
A possible follow-up would be to change the developer/keyboards/_shared/organize-folder-structure.php file to include reference to the shared phphelpfile.md file.

### CODE REVIEW

**Review by mcdurdin — APPROVED** [review]

**Review by mcdurdin — CHANGES_REQUESTED** [review]
The link in step-0 is not correct (one too many `../`!):

```
Getting links from: http://localhost:8888/developer/keyboards/github/step-0
├─BROKEN─ http://localhost:8888/developer/_shared/phphelpfile (HTTP_404)
```

**Comment by DavidLRowe** [comment]
@mcdurdin Sorry to bother you, but I'm groping in the dark trying to figure out what is wrong with the link. I've taken out the extra level, tried adding the `.md` extension (after noticing all the other files in the `_shared` folder were `.php` files). Should I forget about putting it in `_shared` and move it to the `github` folder?

File: developer/keyboards/github/step-0.md [hunk 1]
```diff
@@ -59,6 +59,7 @@ The folder structure for each keyboard has the following components:
 
 `xyz\source\help\`
  : contains the `xyz.php` help file and any resources it references. These files are uploaded to help.keyman.com to provide online help for the keyboard. (The `xyz.php` file often duplicates most of the content of the `welcome.htm` file.)
+ For more information on creating the PHP help file see [Creating a PHP help file](../_shared/phphelpfile.md).
```
Comment (mcdurdin) [inline developer/keyboards/github/step-0.md:62]:
```suggestion
 For more information on creating the PHP help file see [Creating a PHP help file](../_shared/phphelpfile).
```

This should fix it. Sorry for the pain!

Comment (mcdurdin) [reply developer/keyboards/github/step-0.md:62]: I see you fixed it correctly in 4efe5f96fc6d1f41d2ccf7f01296df685e027e03, but that then added phphelpfile to the files to check, and it too has a broken link:

```
Getting links from: http://localhost:8888/developer/keyboards/_shared/phphelpfile
├─BROKEN─ http://localhost:8888/developer/keyboards/_shared/ (HTTP_403)
```

Comment (mcdurdin) [reply developer/keyboards/github/step-0.md:62]: I suggest that we move the phphelpfile.md up to /developer/keyboards/ and link it from there. The _shared folder was used to share content in .php but it's a little harder to do that with Markdown. (Every time we try to remove duplication in documentation, we end up tripping over ourselves somewhere!)

**Comment by DavidLRowe** [comment]
I'll try it, but I think a previous attempt with that text failed (though I've lost track of the permutations at this point).

**Comment by mcdurdin** [comment]
> I'll try it, but I think a previous attempt with that text failed (though I've lost track of the permutations at this point).

Yes, this was due to the issue with having the file in _shared. I'm happy to take over this PR, fix it up and merge if you like.

**Comment by DavidLRowe** [comment]
@mcdurdin That would be great, but I'd like to know what I did wrong (which I guess I can see from the PR when it finally passes the test).

**Comment by mcdurdin** [comment]
OK, I moved phphelpfile.md up a folder, as the problem was with the generated content for the _shared folder -- it doesn't have an index.md or index.php so there were problems with that.

I also tweaked the title and the included code blocks in the file to format them better.

**Review by mcdurdin — APPROVED** [review]

**Comment by mcdurdin** [comment]
Unrelated broken links this time!

```
Getting links from: http://localhost:8888/keyboard/gff_mesobe_fidelat
├─BROKEN─ http://localhost:8888/keyboard/gff_mesobe_fidelat/1.0/gff_mesobe_fidelat-default-1.jpeg (HTTP_404)
├─BROKEN─ http://localhost:8888/keyboard/gff_mesobe_fidelat/1.0/gff_mesobe_fidelat-default-2.jpeg (HTTP_404)
├─BROKEN─ http://localhost:8888/keyboard/gff_mesobe_fidelat/1.0/gff_mesobe_fidelat-default-3.jpeg (HTTP_404)
├─BROKEN─ http://localhost:8888/keyboard/gff_mesobe_fidelat/1.0/gff_mesobe_fidelat-numerals-1.jpeg (HTTP_404)
├─BROKEN─ http://localhost:8888/keyboard/gff_mesobe_fidelat/1.0/gff_mesobe_fidelat-punctuation-1.jpeg (HTTP_404)
├─BROKEN─ http://localhost:8888/keyboard/gff_mesobe_fidelat/1.0/gff_mesobe_fidelat-punctuation-2.jpeg (HTTP_404)
├─BROKEN─ http://localhost:8888/keyboard/gff_mesobe_fidelat/1.0/gff_mesobe_fidelat-tablet-punctuation-1.jpeg (HTTP_404)
```

**Comment by dyacob** [comment]
@mcdurdin I just noticed it also a few minutes back, the fix needed is to add "images/" to the path.  I think a width="100%" attribute is also needed here and with gff_harege_fidelat.  Can I make these changes to the php file without bumping the keyboard version numbers?

**Comment by mcdurdin** [comment]
> Can I make these changes to the php files without bumping the keyboard version numbers?

Yes

**Pull request merged**